release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -1039,6 +1051,11 @@ module openconfig-transport-types {
     description "Ethernet compliance code: 40GBASE_PSM4";
   }
 
+  identity ETH_40GBASE_ACU10M {
+    base ETHERNET_PMD_TYPE;
+    description "Ethernet compliance code: ETH_40GBASE_ACU10M";
```

Comment:
Probably should be ETH_40GBASE_ACU

Reply:
I have doubts in this one... Is it like the previous case and it is IEEE 802.3ba 40GBASE-CR4 ?

Reply:
@dplore 40GBASE_CR4 was already available, so I just update the reference... no need to add a new ACU10M type.